test/fixtures/linter/rules/BestPractices/Negative_1/Component.js

```diff
@@ -0,0 +1,10 @@
+sap.ui.define(["sap/ui/core/UIComponent"], function (UIComponent) {
+	"use strict";
+
+	return UIComponent.extend("com.ui5.troublesome.app.Component", {
+		interfaces: ["sap.ui.core.IAsyncContentCreation"],
```

Comment:
As per https://ui5.sap.com/1.120.12/#/topic/676b636446c94eada183b1218a824717 and https://ui5.sap.com/1.120.12/#/topic/27ce0e4987cd426f8fa3e60836316428 the `interfaces` are defined within the `metadata`.

Reply:
Thanks!